doc/tutorials/introduction/config_reference/config_reference.markdown

```diff
@@ -589,6 +589,12 @@ Some features have been added specifically for automated build environments, lik
 | `OPENCV_CMAKE_HOOKS_DIR` | _empty_ | OpenCV allows to customize configuration process by adding custom hook scripts at each stage and substage. cmake scripts with predefined names located in the directory set by this variable will be included before and after various configuration stages. Examples of file names: _CMAKE_INIT.cmake_, _PRE_CMAKE_BOOTSTRAP.cmake_, _POST_CMAKE_BOOTSTRAP.cmake_, etc.. Other names are not documented and can be found in the project cmake files by searching for the _ocv_cmake_hook_ macro calls. |
 | `OPENCV_DUMP_HOOKS_FLOW` | _OFF_ | Enables a debug message print on each cmake hook script call. |
 
+## Thirdparty modules
```

Comment:
It's better to name it "Math Libraries" or something like that. Eigen and LAPACK should be there too.

Reply:
How about `Contrib Modules`? Since to the best of my knowledge, it's only utilized in opencv_contrib:videostab module. But I agree with the naming for Eigen and LAPACK ...

Reply:
You can add reference to contrib to the option description.

Reply:
Added in 79d03fd25f546257b1942cac1924f9f487bab6fc
There is a bit of an issue though ... can't cross-reference to other sections of the document using anchors. Do Github's markdown engine support id extensions?